packages/backend/src/database/migrations/20240607083146_add_missing_indexes_for_charts_query.ts

```diff
@@ -0,0 +1,43 @@
+import { Knex } from 'knex';
+
+const newTableSingularIndexes = {
+    saved_queries: ['dashboard_uuid'],
+    pinned_chart: ['saved_chart_uuid'],
```

Comment:
https://github.com/lightdash/lightdash/blob/9c09269eef36c9ead531c7e3fa6a4b100257c183/packages/backend/src/models/SavedChartModel.ts#L1178

Reply:
@ZeRego `saved_queries` also doesn't index `saved_query_uuid` you need that too if you want this join to be on two indices

Reply:
nevermind the index is there!